### PR TITLE
Sensor events splitup

### PR DESCRIPTION
**Pull request by gast04**
split up sensorEvents and give every sensor its own service

### CODE REVIEW

File: log.capnp [hunk 1]
```diff
@@ -1910,7 +1910,11 @@ struct Event {
     gpsNMEA @3 :GPSNMEAData;
     can @5 :List(CanData);
     controlsState @7 :ControlsState;
-    sensorEvents @11 :List(SensorEventData);
```
Comment (adeebshihadeh) [inline log.capnp:1913]: in order to keep backwards compatibility with old logs, we don't remove fields, but add `DEPRECATED` at the end of the name and move it to the bottom of the struct with the other deprecated fields

File: services.py [hunk 2]
```diff
@@ -21,7 +21,11 @@ def __init__(self, port: int, should_log: bool, frequency: float, decimation: Op
 services = {
   # service: (should_log, frequency, qlog decimation (optional))
   # note: the "EncodeIdx" packets will still be in the log
-  "sensorEvents": (True, 100., 100),
+  "gyroscope": (True, 100., 100),
+  "accelerometer": (True, 100., 100),
```
Comment (adeebshihadeh) [inline services.py:25]: aren't these 104Hz?

File: log.capnp [hunk 3]
```diff
@@ -1949,6 +1949,13 @@ struct Event {
     can @5 :List(CanData);
     controlsState @7 :ControlsState;
     sensorEvents @11 :List(SensorEventData);
```
Comment (adeebshihadeh) [inline log.capnp:1951]: add the DEPRECATED suffix and move it down with the other deprecated events
```suggestion
    sensorEventsDEPRECATED @11 :List(SensorEventData);
```

File: services.py [hunk 4]
```diff
@@ -22,6 +22,13 @@ def __init__(self, port: int, should_log: bool, frequency: float, decimation: Op
   # service: (should_log, frequency, qlog decimation (optional))
   # note: the "EncodeIdx" packets will still be in the log
   "sensorEvents": (True, 100., 100),
+  "gyroscope": (True, 100., 100),
```
Comment (adeebshihadeh) [inline services.py:25]: aren’t some of these 104Hz?

Comment (gast04) [reply services.py]: yes, gyro and accel are 104Hz, just thought as we in general aim for 100Hz, changed

File: log.capnp (hunk 3 above)
Comment (gast04) [reply log.capnp]: done

File: services.py [hunk 5]
```diff
@@ -21,7 +21,14 @@ def __init__(self, port: int, should_log: bool, frequency: float, decimation: Op
 services = {
   # service: (should_log, frequency, qlog decimation (optional))
   # note: the "EncodeIdx" packets will still be in the log
-  "sensorEvents": (True, 100., 100),
+  "sensorEventsDEPRECATED": (True, 100., 100),
```
Comment (adeebshihadeh) [inline services.py:24]: deprecated services don't need to be here, just need to be kept in the cereal structs

File: services.py [hunk 6]
```diff
@@ -21,7 +21,14 @@ def __init__(self, port: int, should_log: bool, frequency: float, decimation: Op
...
+  "gyroscope": (True, 104., 100),
+  "gyroscope2": (True, 100., 100),
+  "accelerometer": (True, 104., 100),
```
Comment (adeebshihadeh) [inline services.py:27]: update the qlog decimation to be 104 as well

File: services.py (hunk 5 above)
Comment (gast04) [reply services.py]: updated

File: services.py (hunk 6 above)
Comment (gast04) [reply services.py]: updated

**Review by adeebshihadeh — APPROVED** [review]

File: services.py [hunk 7]
```diff
@@ -21,7 +21,13 @@ def __init__(self, port: int, should_log: bool, frequency: float, decimation: Op
 services = {
   # service: (should_log, frequency, qlog decimation (optional))
   # note: the "EncodeIdx" packets will still be in the log
-  "sensorEvents": (True, 100., 100),
```
Comment (adeebshihadeh) [inline services.py:24]: also need to add this back

**Pull request merged**